src/requestStub.js

```diff
@@ -65,6 +66,49 @@ getJasmineRequireObj().AjaxRequestStub = function() {
       return this.action === CALL;
     };
 
+    this.andReturnStream = function(options) {
+      if (!options) {
+        options = {};
+      }
+
+      this.action = STREAM;
+      this.status = options.status || 200;
+
+      this.contentType = options.contentType;
+      this.responseHeaders = options.responseHeaders;
+      this.responseURL = options.responseURL;
+
+      this.streamController = {
+        data: [],
+        requestCancelled: false,
+        requestFinished: false,
+
+        sendData: function(text) {
+          this.data.push(text);
+        },
+        finishRequest: function() {
```

Comment:
These functions get overridden by the `fakeRequest` when the request is actually started, so the errors probably don't get thrown.

Reply:
```js
var controller = (...).andReturnStream();
controller.finishRequest();
controller.cancelRequest(); // and here we should get an Error

var req = new XmlHttpRequest();
```

In this case I should get an error.